packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -52,22 +52,29 @@ class CustomFormatter(logging.Formatter):
     2. Overrides 'filename' with the value of 'file_name_override', if it exists.
     """
 
+    def __init__(self, fmt, color_log_enabled=False):
```

Comment:
Annotate arguments when possible

e.g. ``def __init__(self, fmt; str, color_log_enabled: bool =False)``


If you do so, there is no need to annotate ``self.color_log_enabled``